docs/how-to/extending/builtin.qmd

```diff
@@ -169,3 +172,60 @@ con.execute(cardinality("foo"))
 
 Here, Snowflake is informing us that the `ARRAY_SIZE` function does not accept
 strings as input.
+
+
+## Aggregate functions
```

Comment:
I think a few sentences of description would be useful to set the scene here before an example. Copy-pasting and slight edits from the scalar docs above:

```
Aggregate functions that aren't exposed in ibis directly can be accessed using the
`@ibis.udf.agg.builtin` decorator.

::: {.callout-tip}
### [Ibis APIs](../../reference/index.qmd) may already exist for your function.

Builtin aggregate UDFs are designed to be an escape hatch when Ibis doesn't have
a defined API for a built-in database function.

See [the reference documentation](../../reference/index.qmd) for existing APIs.
:::
```

Reply:
Put the above text in verbatim, thanks!